SimTKcommon/Simulation/include/SimTKcommon/internal/StateImpl.h

```diff
@@ -158,50 +233,63 @@ class CacheEntryInfo {
     CacheEntryInfo& deepAssign(const CacheEntryInfo& src) {
         assert(src.isReasonable());
```

Comment:
Perhaps `isReasonable()` could have a more descriptive name? I'm unfamiliar with this method and wasn't able to find its implementation [by searching the GitHub repo](https://github.com/simbody/simbody/search?utf8=%E2%9C%93&q=isReasonable).